app/components/candidate_interface/rejection_reasons_component.rb

```diff
@@ -31,5 +29,15 @@ def rejection_reasons_row(application_choice)
         value: application_choice.rejection_reason,
       }
     end
+
+    def course_details_row_value(application_choice)
+      if EndOfCycleTimetable.find_down?
+        "#{application_choice.offered_course.name_and_code} <br> #{application_choice.course.description}".html_safe
```

Comment:
Do we know if the course data from Find is adequately sanitised? If not, we may need to rethink how to display this so that we aren't marking the entire string safe.

Reply:
Provider and Course codes are generated on the backend. I'm pretty certain it's not possible to change them via the UI.

Reply:
Same with course descriptions?